### DISCUSSION TITLE
Few questions post Doks installation

### DISCUSSION THREAD

**Discussion by rajtilakjee** [thuliteio/doks#1007] [Q&A] [answered]
I was able to successfully install Doks on my website https://developer-roadmaps.netlify.app/. However, I do have a few questions that might seem stupid because I am a complete novice when it comes to web development. So please bear with me.

1. Why are there 6 different branches on my Github repo (https://github.com/rajtilakjee/developer-roadmaps) and what should I do with them?
2. Why does it say that my main branch isn't protected and what should I do about it?
3. I want to rename the Docs folder to something more meaningful for my project like "Skill-based Roadmaps" and "Role-based Roadmaps". How should I go about it?

These are the only ones I could think of right now. Any help would be appreciated. Thanks.

**Comment by h-enk** [comment, marked as answer]
> However, I do have a few questions that might seem stupid because I am a complete novice when it comes to web development. So please bear with me.

No worries. There are no stupid questions 😉 Thanks for asking!


> Why are there 6 different branches on my Github repo (https://github.com/rajtilakjee/developer-roadmaps) and what should I do with them?

Good point! You don't need them. You should also delete the cloned `.git` file. It's because of the `git clone` command. Using a tool like [degit](https://github.com/Rich-Harris/degit) to scaffold your project would be better:

```bash
npx degit h-enk/doks my-doks-site
```

Note that, this issue will no longer be relevant with Doks 1.0 (available soon), it uses a scaffold project structure to set up your Doks project.

> Why does it say that my main branch isn't protected and what should I do about it?

This is only relevant when you're working in a team on your Doks project. You could set up branch protection rules (e.g. preventing a collaborator on your project to push changes directly) — thereby safeguarding your code in a branch. Learn more [about protected branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches), So, you could dismiss the message.

> I want to rename the Docs folder to something more meaningful for my project like "Skill-based Roadmaps" and "Role-based Roadmaps". How should I go about it?

Just rename it manually or by using a tool like e.g. [slugify-cli](https://github.com/sindresorhus/slugify-cli):

```bash
npx slugify-cli "Skill-based Roadmaps"
```

Result `skill-based-roadmaps`

**Comment by rajtilakjee** [comment]
Thanks so much for replying to my queries. I didn't really expect it, that too so soon since I am pretty sure you are a busy person.
